### ISSUE TITLE
Ignore All Pattern not respected on tab switching

### ISSUE THREAD

**Issue by orangedrank** [ForkIssues/TrackerWin#898]
When I use the ignore all *pattern* feature in one tab it works. If I switch to another tab on a different repository it will stop respecting it (even if it is already in the ignore list).

**Comment by DanPristupov** [comment]
Do you mean the `.gitignore` file? It's per repository only.

If you really need a global git ignore, you can find more info here: https://stackoverflow.com/questions/7335420/global-git-ignore#7335487.

**Comment by orangedrank** [comment]
Yes, the .gitignore. It's not respecting stuff already on it's list in the other repositories after switching tabs. Very odd.
